script.js

```diff
@@ -1,8 +1,122 @@
+    liveUrl: '#',
+    sourceUrl: '#',
+  },
+  {
+    id: 3,
+    title: 'Profesional Art Printing Data',
+    listArray: ['HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/images/Card-Placeholder.png',
+    imageAlt: 'Project image with a vertical dark gradient',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard.',
+    liveUrl: '#',
+    sourceUrl: '#',
+  },
+  {
+    id: 4,
+    title: 'Profesional Art Printing Data',
+    listArray: ['HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/images/Card-Placeholder.png',
+    imageAlt: 'Project image with a vertical dark gradient',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard.',
+    liveUrl: '#',
+    sourceUrl: '#',
+  },
+  {
+    id: 5,
+    title: 'Profesional Art Printing Data',
+    listArray: ['HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/images/Card-Placeholder.png',
+    imageAlt: 'Project image with a vertical dark gradient',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard.',
+    liveUrl: '#',
+    sourceUrl: '#',
+  },
+  {
+    id: 6,
+    title: 'Profesional Art Printing Data',
+    listArray: ['HTML', 'Bootstrap', 'Ruby'],
+    image: 'assets/images/Card-Placeholder.png',
+    imageAlt: 'Project image with a vertical dark gradient',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry\'s standard.',
+    liveUrl: '#',
+    sourceUrl: '#',
+  },
+];
+const featuredContainer = document.querySelector('.featured-work');
+const cardsContainer = document.querySelector('.card-works');
+
+worksArray.forEach((work) => {
+  if (work.id === 0) {
+    featuredContainer.insertAdjacentHTML('beforeend', `
+    <div class="featured" data-card="${work.id}">
+      <img src="${work.image}" alt="${work.imageAlt}">
+      <div class="right-block">
+        <h3>${work.title}</h3>
+        <p>${work.description}</p>
+        <ul class="tags">
+        <li>${work.listArray[0]}</li>
+        <li>${work.listArray[1]}</li>
+        <li>${work.listArray[2]}</li>
+        <li>${work.listArray[3]}</li>
+        </ul>
+        <button class="btn featured-btn work-btn" data-id="${work.id}">See Project</button>
+      </div>
+    </div>`);
+  } else {
+    cardsContainer.insertAdjacentHTML('beforeend', `
+    <div class="card" data-card="${work.id}">
+      <img src="${work.image}" alt="${work.imageAlt}">
+      <div class="right-block">
+        <h3>${work.title}</h3>
+        <p>${work.description}</p>
+        <ul class="tags">
+        <li>${work.listArray[0]}</li>
+        <li>${work.listArray[1]}</li>
+        <li>${work.listArray[2]}</li>
+        </ul>
```

Comment:
Brilliant job using the  `insertAdjacentHTML`! But let's fix our tech stack tags. Currently it looks undefined. And please fix the overflowing tag as I marked below on the image.
<img width="374" alt="Screenshot 2023-02-01 at 11 07 23 PM" src="https://user-images.githubusercontent.com/25887253/216152728-2399fcd5-4f91-4382-9d95-1bbf6c2b8b47.png">